packages/devtools_app/lib/src/screens/debugger/program_explorer_model.dart

```diff
@@ -305,22 +303,23 @@ class VMServiceObjectNode extends TreeNode<VMServiceObjectNode> {
   }
 
   Future<void> populateLocation() async {
-    ScriptRef script = this.script;
-    int tokenPos = 0;
+    // ignore: unused_local_variable
+    ScriptRef? scriptRef = script;
```

Comment:
this smells a bit. We aren't using scriptRef everywhere in this method where we previously were using script. If we use scriptRef everywhere we were previously using script, we should be able to get rid of this ignore

Reply:
Oops! Ignore is now gone. We were setting a variable with type `ScriptRef` to type `Script`. Refactored so they are now separate variables.